protoutils/messages.go

```diff
@@ -56,10 +56,12 @@ func ConvertOrientationToProto(o spatialmath.Orientation) *commonpb.Orientation
 	oo := &commonpb.Orientation{}
 	if o != nil {
 		ov := o.OrientationVectorDegrees()
-		oo.OX = ov.OX
-		oo.OY = ov.OY
-		oo.OZ = ov.OZ
-		oo.Theta = ov.Theta
+		if ov != nil {
+			oo.OX = ov.OX
+			oo.OY = ov.OY
+			oo.OZ = ov.OZ
+			oo.Theta = ov.Theta
+		}
```

Comment:
[q] do we populate a default anywhere? An all zero orientation vector or an all NaN orientation vector would be valid since it will cause the spatialmath package to error.